.github/workflows/comment-pr-on-check-failure.yml

```diff
@@ -15,10 +15,60 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  skip_check:
+    name: Job to check if the workflow ${{ github.event.workflow.name }} can be skipped
+    runs-on: ubuntu-latest
```

Comment:
add this here` if: github.event.workflow_run.event == 'pull_request' ||  github.event.workflow_run.event == 'pull_request_target'`